emitter.ts

```diff
@@ -0,0 +1,86 @@
+interface EventCallback {
```

Comment:
A simple event emitter where on/once return a function that removes exactly that listener.